recipes/chemicalfun/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -u
+ferr(){
+    echo "$@"
+    exit 1
+}
+
+mkdir -p build
+cd build
+
+# Configure step
+cmake -DPYTHON_EXECUTABLE:FILEPATH="$PYTHON" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+      -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
+      ..
+# Build step
+make -j${CPU_COUNT}
+make install
```

Comment:
Is there a `make check` or `ctest` that we can run here?

Reply:
@ocefpaf Thanks for the review. The only tests I added are in the meta.yaml, checking that the files are installed in the right place and that the python module imports correctly. If necessary, I can try to add additional tests using ctest, or from python pytest.

Reply:
If those are not available from upstream we don't really require those here. I'll merge it and defer to your judgment in the feedstock.